src/main/java/net/starlegacy/explosionregen/package-info.java

```diff
@@ -1,4 +0,0 @@
-@ParametersAreNonnullByDefault
```

Comment:
Why is this removed?

Reply:
What is it doing that helps

Reply:
It makes it so that parameters are non null by default.